### PR TITLE
Font Copy and Revving Fix

### LINKED ISSUE #116: Handle web fonts (copy task and revving)
:point_up:

### PR DESCRIPTION
**Pull request by solomonhawk**
Fixes #116

### CODE REVIEW

File: gulpfile.js/tasks/rev/rev-font-workaround.js [hunk 1]
```diff
@@ -54,3 +58,11 @@ gulp.task('rev-font-workaround', ['rev-assets'], function() {
 
   return merge.apply(this, streams);
 });
+
+function exists(path) {
```
Comment (solomonhawk) [inline gulpfile.js/tasks/rev/rev-font-workaround.js]: Could put this in a util.

File: gulpfile.js/tasks/rev/rev-font-workaround.js [hunk 2]
```diff
@@ -54,3 +58,11 @@ gulp.task('rev-font-workaround', ['rev-assets'], function() {
...
+  try {
+    return fs.openSync(filePath, 'r')
+  } catch(e) {
+    return false
+  }
+}
```
Comment (greypants) [inline gulpfile.js/tasks/rev/rev-font-workaround.js]: I don't think this is quite the right approach. The only thing this task should be doing is copying the hash off of the iconFont.svg file, and adding it to the corresponding woff, eot, and ttf icon font files. I'd leave this file as is, and instead, be more specifc in the "not these" list in `rev-assets.js`, allowing non-iconFont files to be revved.

File: gulpfile.js/tasks/rev/rev-assets.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ var rev    = require('gulp-rev');
 // 1) Add md5 hashes to assets referenced by CSS and JS files
 gulp.task('rev-assets', function(){
   // See comment below about eot,woff, and ttf
-  var notThese = '!' + config.publicDirectory + '/**/*+(css|js|json|eot|woff|ttf|html)'
+  var notThese = '!' + config.publicDirectory + '/**/*+(css|js|json|eot|woff|woff2|ttf|html)'
```
Comment (greypants) [inline gulpfile.js/tasks/rev/rev-assets.js]: See note below.

Comment (solomonhawk) [inline gulpfile.js/tasks/rev/rev-assets.js]: Assumed woff2 had the same implications as ttf/eot/woff

**Comment by greypants** [comment]
Moved this to #119 and added a commit

**Pull request closed without merging**